### PR TITLE
feat: replace select picker v3

### PR DESCRIPTION
**Pull request by shivani170**
# Description
-  ExternalLinks.component  -> More than 1 spectic appliv=cation External link 
- [x] Devtron Apps > App Details > Pods
- [x] Devtron Apps > App Details > Container
Image
- [x] ~ExternalLinks.Filters> 1 Link (nodeLevelExternalLink)~ It is cluster/application dropdown with apply select
Image

- [ ] ~ConfigureLinkAction~ Click on Add external link > It is Icons with url. Need to do some devImage

- [x] GitProvider =>  Global Config > Git Account >  Git Host 
Image

- [x] SESConfigModal => Global Config  > Notification > AWS region
Image

- [ ] ~ClusterSelector~ (Will be picking Header selector later)
- [ ] ~Sidebar~ (Will be picking Highlight search selector later)
- [x] Security Policy Edit
Image
- [x] ~Update Severity~ (Deleted the unused component)
- [x] ~Vulnerability Exposure~ (Arun will pick whole security Vulnerability issues)
- [x] EphemeralContainerDrawer > Target Container Name
Image

- [x] terminal.utils.tsx
    - [x] Container
    - [x] Terminal shell type 
  Image
-[ ] ~ChartValueEditor~ Will be doing in v4 due to version complexity
Image
- ChartVAlueViewComponent => Charts > Discover > Preset Values 
- [x] Project
- [x] Deploy to enviroment
Image

- [x] ProjectUpdateModal.tsx => Chart Value > Chart Update > Project edit
Image

Image

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/gitProvider/GitProvider.tsx [hunk 1]
```diff
@@ -635,6 +636,20 @@ const GitForm = ({
         )
     }
 
+    const renderGitHostBottom = () => {
+        return (
+            <div
+                className="flex left pl-10 pt-8 pb-8 cb-5 cursor bcn-0 dc__react-select__bottom dc__border-top "
+                onClick={(selected) => {
```
Comment (RohitRaj011) [inline src/components/gitProvider/GitProvider.tsx:643]: remove selected

File: src/components/gitProvider/GitProvider.tsx [hunk 2]
```diff
@@ -635,6 +636,20 @@ const GitForm = ({
         )
     }
 
+    const renderGitHostBottom = () => {
+        return (
+            <div
```
Comment (RohitRaj011) [inline src/components/gitProvider/GitProvider.tsx:641]: use button with dc__transparent

File: src/components/security/security.types.ts [hunk 3]
```diff
@@ -290,3 +290,37 @@ export interface FetchPolicyQueryParams {
...
+
+export interface AddCveModalState {
+    view: string
+    cve: string
+    policy: VulnerabilityAction
+    clusters: {
+        name: string
+        policy: VulnerabilityAction
+        isCollapsed: boolean
+        environments: [
```
Comment (RohitRaj011) [inline src/components/security/security.types.ts:302]: break down this type

File: src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetail.component.tsx [hunk 4]
```diff
@@ -105,7 +105,7 @@ const NodeDetailComponent = ({
 
     const selectedContainerValue = isResourceBrowserView ? selectedResource?.name : podMetaData?.name
     const _selectedContainer = selectedContainer.get(selectedContainerValue) || containers?.[0]?.name || ''
-    const [selectedContainerName, setSelectedContainerName] = useState(_selectedContainer)
+    const [selectedContainerName, setSelectedContainerName] = useState(({label: _selectedContainer, value: _selectedContainer}))
```
Comment (RohitRaj011) [inline src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetail.component.tsx:108]: add type

File: src/components/v2/appDetails/k8Resource/nodeType/Node.component.tsx [hunk 5]
```diff
@@ -26,13 +26,14 @@ import NodeDeleteComponent from './NodeDelete.component'
...
 import { useSharedState } from '../../../utils/useSharedState'
-import { NodeLevelExternalLinks } from '../../../../externalLinks/ExternalLinks.component'
+import { getExternalLinkIcon, NodeLevelExternalLinks } from '../../../../externalLinks/ExternalLinks.component'
 import { OptionTypeWithIcon } from '../../../../externalLinks/ExternalLinks.type'
 import { getMonitoringToolIcon } from '../../../../externalLinks/ExternalLinks.utils'
 import { NoPod } from '../../../../app/ResourceTreeNodes'
 import './nodeType.scss'
 import { ReactComponent as DropDown } from '../../../../../assets/icons/ic-dropdown-filled.svg'
 import { getPodRestartRBACPayload } from '../nodeDetail/nodeDetail.api'
+import { start } from 'repl'
```
Comment (RohitRaj011) [inline src/components/v2/appDetails/k8Resource/nodeType/Node.component.tsx:36]: remove

File: src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx [hunk 6]
```diff
@@ -471,19 +448,14 @@ export const ChartProjectSelector = ({
 }: ChartProjectSelectorType): JSX.Element => {
     return (
         <label className="form__row form__row--w-100 fw-4">
```
Comment (RohitRaj011) [inline src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx:450]: make div

File: src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx [hunk 7]
```diff
@@ -471,19 +448,14 @@ export const ChartProjectSelector = ({
...
-                components={{
-                    IndicatorSeparator: null,
-                    DropdownIndicator,
-                    Option,
-                }}
+            <SelectPicker
+                inputId="project-select"
+                required
+                name="project"
+                label="Select Project"
```
Comment (RohitRaj011) [inline src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx:455]: this should be Project pls verify

**Review by RohitRaj011 — DISMISSED** [review]

**Review by RohitRaj011 — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**